wcfsetup/install/files/lib/system/language/preload/RegisterPreloadPhrases.class.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace wcf\system\language\preload;
+
+use wcf\data\language\Language;
+use wcf\system\event\IEvent;
+
+final class RegisterPreloadPhrases implements IEvent
```

Comment:
The event namespace is especially important here, because the name does not make it clear that this is an event. It could also be a *command*.

In this case it's hard to choose an appropriate name, because this event is not about something that *happened* (“User Logged In”, “Cache was cleared”, “Application booted successfully”), but that *will happen*.

The Microsoft style guide for C# appears to use the Gerundium for events like these: https://stackoverflow.com/a/724375/782822

How about: `CollectingPreloadedPhrases`.